pkg/test/resourcefixture/testdata/basic/colab/v1alpha1/colabruntime/basiccolabruntime/_http.log

```diff
@@ -331,38 +443,26 @@ X-Xss-Protection: 0
 
 {
   "createTime": "2024-04-01T12:34:56.123456Z",
-  "dataPersistentDiskSpec": {
-    "diskSizeGb": "100",
-    "diskType": "pd-standard"
-  },
   "displayName": "colabruntime ${uniqueId}",
-  "eucConfig": {},
   "expirationTime": "2024-09-01T12:34:56.123456Z",
   "healthState": 1,
   "idleShutdownConfig": {
     "idleTimeout": "10800s"
   },
   "labels": {
     "aiplatform.googleapis.com/colab_enterprise_pool": "false",
-    "aiplatform.googleapis.com/notebook_runtime_gce_instance_id": "6515919734255920475"
-  },
-  "machineSpec": {
```

Comment:
nit: maybe try to match these? My guess is that these values are taken from the ColabRuntimeTemplate during the creation.

Reply:
Unfortunately, machineSpec is not a field supported in the version of the lib that our mockgcp is dependent on: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/v1.130.2/mockgcp/generated/mockgcp/cloud/aiplatform/v1beta1/notebook_runtime.pb.go#L450